packages/cursorless-vscode/extension.ts

```diff
@@ -64,6 +55,11 @@ export async function activate(
     injectIde(vscodeIDE);
   }
 
+  const commandServerApi =
+    vscodeIDE.runMode === "test"
+      ? getFakeCommandServerApi()
+      : await getCommandServerApi();
```

Comment:
Note that during testing we have a proper fake command server api now, which is used by [`prePhraseSnapshot.test.ts`](https://github.com/cursorless-dev/cursorless/pull/1280/files#diff-aa3d46ba59620c5260c9095f28685e60b9ed868f6092b68038030f6f39e2e13a)